sections/security/requestpayloadsizelimit.md

```diff
@@ -0,0 +1,60 @@
+# Limit payload size using a reverse-proxy or a middleware
+
+### One Paragraph Explainer
+
+When handling incoming requests in your web application, you should limit the size of their respective payloads. Incoming requests with
+unlimited body/payload sizes can lead to your application crashing due to a denial-of-service outage or other unwanted side-effects.
+Many popular middleware-solutions for parsing request bodies, such as the already-included `body-parser` package for express, expose
+options to limit the sizes of request payloads, making it easy for developers to implement this functionality. You can also
+integrate a request body size limit in your reverse-proxy/web server software if supported. Below are examples for limiting request sizes using
+`express` and/or `nginx`.
+
+### Example code for `express`
+
+```javascript
+const express = require('express');
+
+const app = express();
+
+const parseJsonBody = express.json({ limit: '300kb' }); // body-parser defaults to a body size limit of 100kb
```

Comment:
because you'd naturally apply this middleware to all routes I will give a better example of configuring bodyParser for this limit, or at the very least if you create a middleware like that then I'd put it in `app.use()` instead of every route.

Reply:
@BrunoScheufler I second @lirantal advice

Reply:
Agreed, I'll simply use the JSON body parsing middleware in the app, instead of every request. Just thought the additional work of parsing the body for every route might be used in a more customizable way where you could select routes to parse the body for, instead of every route with an application/json content type.